spec/features/login/second_login_form_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature 'Login Page', type: :feature do
+  background { visit(new_user_session_path) }
+
+  describe 'Submit button' do
+    given(:user) { create(:user) }
+
+    scenario 'without email and password' do
+      click_button 'Log in'
+      expect(page.has_content?('Invalid Email or password.')).to be_truthy
+    end
+    scenario 'with invalid email and password' do
+      within 'form' do
+        fill_in 'user[email]', with: 'unknown@test.com'
+        fill_in 'user[password]', with: 'fakepassword'
+      end
+      click_button 'Log in'
+      expect(page.has_content?('Invalid Email or password.')).to be_truthy
+    end
+
+    scenario 'with valid email and password' do
+      within 'form' do
+        fill_in 'user[email]', with: user.email
+        fill_in 'user[password]', with: user.password
+      end
+      click_button 'Log in'
```

Comment:
- [ ] Optional suggestion. You can also create the `third_login_form_spec` with the following implementation

```
 describe 'the signin process', type: :feature do
     before :each do
         User.create(email: 'user1@gmail.com', password: 'password', name: 'User1')
     end

     it 'signs @user in' do
         visit '/users/sign_in'
         fill_in 'Email', with: 'user1@gmail.com'
         fill_in 'Password', with: 'password'
         click_button 'Log in'
         expect(current_path).to eq(root_path)
         expect(page).to have_text('Signed in successfully.')
     end

     it 'does not sign in with wrong info' do
         visit '/users/sign_in'
         fill_in 'Email', with: 'user1@gmail.com'
         fill_in 'Password', with: 'mypassword'
         click_button 'Log in'
         expect(page).to have_text('Invalid Email or password.')
     end
 end
```

Reply:
Hi @Emmyn5600 
Thank you for this great suggestion